app/src/main/java/com/framgia/wsm/utils/Constant.java

```diff
@@ -39,4 +39,19 @@ private Constant() {
         public static final String LEAVE_EARLY_WOMAN_M = "Leave early (woman only) (M)";
         public static final String LEAVE_EARLY_WOMAN_A = "Leave early (woman only) (A)";
     }
+
+    public static class Time {
+        public static int SEVEN_HOUR = 7;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName